README.md

```diff
@@ -6,22 +6,37 @@
 
 Interaction Kit is a framework for creating Discord slash command bots over the HTTP API.
 
-<!-- ## HTTP vs Gateway
+## HTTP vs Gateway
 
-In short, you probably should be using [discord.js]() unless you know why you're here.
+**TL;DR**: you probably should be using [discord.js] unless you know why you're here.
 
-**Gateway**:
+There are two ways for bots to recieve events from Discord.  Most API wrappers such as [discord.js] use a WebSocket connection (the "Gateway") to receive events, but Interaction Kit receieves Interaction events over HTTP.  As such, there are some major points to keep in mind before deciding to use Interaction Kit.
 
-**HTTP**:
+### Gateway
 
-Interaction Kit **does not** interface with the Gateway, which limits information is available to your application (and also means you can never be sure the information you have is up to date - if you request information about a User, and that user then updates their profile, if you are on Gateway you'll get a notification of the update event. With the HTTP API, you would not know the data had been changed).
+The Gateway is a WebSocket that a bot maintains with Discord whenever it is online, giving it events as they happen.  The bot is able to subscribe to a wide array of events through Gateway Intents, allowing it to be notified on events such as message sends, a member changing their nickname or a guild changing its icon.
 
-There are other drawbacks as well - HTTP-only bots don't currently show up as Online in a Guild's member list, which will lead to a lot of questions about whether the application is working or not.
+This is the only way to receive the majority of events Discord provides, and when dealing with users or guilds you can generally be sure you are operating on current information with sufficient caching.
 
+The main drawback of the Gateway is that it requires your bot to maintain a connection with Discord at all times.  While there are affordable hosts for smaller scale bots, scaling can become expensive quickly and some developers have very low budgets.
+
+### HTTP
+
+HTTP-only bots are sent Interaction events through HTTP requests sent from Discord.  These generally have a higher latency than a WebSocket connection from the same host and cannot listen for other types of event.
```

Comment:
Perhaps consider putting Interaction in quotes to set it apart as a new term rather than what might seem like a fancy name for an event for Interaction Kit?

Reply:
I think a better option would be
```diff
- WebSocket connection (the "Gateway")
+ WebSocket connection called a Gateway
```
As that would make all references to Discord concepts consistent with each other.